### ISSUE TITLE
Merge editor: checkbox menu feedback

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#153185]
<img width="255" alt="image" src="https://user-images.githubusercontent.com/900690/175760224-1e5f73d1-6755-42e3-b467-77fee714f833.png">

I believe the top 4 entries are mutually exclusive so I would have expected a radio button style.

The last one seems to be a toggle, so a checkbox is fine but should probably put a separator to the top.

//cc @daviddossett

**Comment by bpasero** [comment]
I think a similar issue is in the new layout menu where I would have expected radio button:

<img width="225" alt="image" src="https://user-images.githubusercontent.com/900690/175760247-2f892482-3c4c-49ee-8e3b-80b3d2d169b6.png">

**Comment by hediet** [comment]
The menu item only has a "checked" property. How can I get a radio button there?
Do we have a context menu that has this?

**Comment by bpasero** [comment]
Yeah good point, I think macOS does not allow for radio button in menus. At least separators would help.

**Comment by hediet** [comment]
There is now a separator:

![image](https://user-images.githubusercontent.com/2931520/178754704-c114ca67-2ffd-476d-8c92-30992f450335.png)
